expression/builtin_math_test.go

```diff
@@ -115,32 +116,65 @@ func (s *testEvaluatorSuite) TestFloor(c *C) {
 
 	sc := s.ctx.GetSessionVars().StmtCtx
 	tmpIT := sc.IgnoreTruncate
-	sc.IgnoreTruncate = true
```

Comment:
yeah,
but we can keep this line,
thus `IgnoreTruncate` will always be true for all the test case in this functon.